README.md

```diff
@@ -1,45 +1,231 @@
+The **_library package_** can be used independently of the **_operator_** and is consumed as a helper library by other modules in the Kyma ecosystem.
 
-**Disclaimer**: This repository is still in development and implementation details can change quickly.
+### Contents
 
-## Local setup
+* [Stability](#stability)
+* [Operator specification](#operator-specification)
+  * [Manifest Custom Resource](#manifest-custom-resource)
+  * [Sample resource](#sample-resource)
+* [Manifest library](#manifest-library)
+  * [Sample usage](#sample-usage)
+* [Run the operator](#run-the-operator)
+  * [Local setup](#local-setup)
+  * [Cluster setup](#cluster-setup)
+* [Contribution](#contribution)
+* [Versioning and releasing](#versioning-and-releasing)
 
-- Spin up a k3d cluster (other types like kind are also supported)
-- From the **api** folder run: 
-`make install`
-- From the **operator** folder run
-  `make build` 
-followed by
-  `make run`
-to run the operator
-- Config your own **kubeconfig** by setting the **r.Config** variable inside */operator/controllers/manifest_controller.go*
+## Stability
 
+The architecture and implementation are majorly based on proof-of-concept (POC) results and can change rapidly as we work towards providing a stable solution.
+In general, the reconciliation and resource processing framework is considered stable and ready to be used.
 
-## Use manifest packages in your own operator
+As part of this repository the following components are offered:
 
-- Import package `"github.com/kyma-project/module-manager/operator/pkg/manifest"`
-- [Package Specifications](https://pkg.go.dev/github.com/kyma-project/module-manager/operator/pkg/manifest) 
-- Package usage example:
+| System Component                                          | Stability                                                                                         |
+|-----------------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| [Manifest](operator/api/v1alpha1/manifest_types.go)       | Alpha-Grade - do not rely on automation and watch upstream as close as possible                   |
+| [Controller](operator/controllers/manifest_controller.go) | In active development - expect bugs and fast paced development                                    |
+| [Library](operator/pkg)                                   | In active development - expect bugs and fast paced development. Detailed documentation to follow. |
+
+## Operator specification
+
+The module-manager reconciles the `Manifest` custom resource.
+As observed in the [Sample Manifest CR](operator/config/samples/operator_v1alpha1_manifest.yaml) and [API definition](operator/api/v1alpha1/manifest_types.go), 
+the `Spec` contains the necessary properties to process installations.
+
+### Manifest Custom Resource
```

Comment:
We use [sentence case](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#sentence-case) in headlines. 

```suggestion
### Manifest custom resource
```